src/pages/userManagement/utils.js

```diff
@@ -0,0 +1,28 @@
+export function formatUserMessage(user, intl) {
```

Comment:
I think this file should have a more illustrative name than utils.js. I don't love relying on directory structure for context.